WalletWasabi/WabiSabi/Models/ConnectionConfirmationRequest.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Crypto.ZeroKnowledge;
+using WalletWasabi.WabiSabi.Crypto;
+
+namespace WalletWasabi.WabiSabi.Models
+{
+	public class ConnectionConfirmationRequest
+	{
+		public ConnectionConfirmationRequest(Guid roundId, Guid aliceId, RegistrationRequestMessage zeroCredentialRequests, RegistrationRequestMessage realCredentialRequests)
```

Comment:
Idk how this request will be serialized but it looks like it is leaking which credentials are zero value and which are not. The idea is that nobody, except the client, should know that. 
```suggestion
		public ConnectionConfirmationRequest(Guid roundId, Guid aliceId, RegistrationRequestMessage credentialRequests)
```